### PR TITLE
Add cpu load,tcp\udp usage statistics in prom `/metrics` endpoint.

### PR DESCRIPTION
**Pull request by Colstuwjx**
Hi folks,

Related to #1670  , I'd like to take a PR to expose more about `container_cpu_load_average_10s`, `container_network_tcp_usage_total`, `container_network_udp_usage_total` metric within the `/metrics` endpoint.

Currently, tcp/udp usage will be disabled via `disable_metrics` flag by default, thus `/metrics` would show zero values. Due to [ContainerSpec](https://github.com/google/cadvisor/blob/master/info/v2/container.go#L67) has no `ignoreMetrics` flag, I don't know how to handle this...

Thanks.

### CODE REVIEW

**Comment by Colstuwjx** [comment]
Ping @dashpole  @jimmidyson

**Comment by dashpole** [comment]
@k8s-bot ok to test

**Comment by dashpole** [comment]
> Currently, tcp/udp usage will be disabled via disable_metrics flag by default, thus /metrics would show zero values. Due to ContainerSpec has no ignoreMetrics flag, I don't know how to handle this...

Sorry, why would you want the container spec to have ignoreMetrics?

**Comment by Colstuwjx** [comment]
Yeah, I mean that due to no ignoreMetrics in ContainerSpec, I cannot determine whether the metric is disabled. In the code level, ignoreMetrics should not appeared in that, so, just expose zero values if they are disabled...

**Comment by Colstuwjx** [comment]
@dashpole  Hi, I have made things clear, pls let me know if there is something need to change.

File: info/v2/container.go [hunk 1]
```diff
@@ -262,6 +262,22 @@ type TcpStat struct {
 	Closing     uint64
 }
 
+func (t *TcpStat) TcpStatMap() map[string]uint64 {
```
Comment (dashpole) [inline info/v2/container.go]: Please remove this.  It isnt used anywhere

File: info/v1/container.go [hunk 2]
```diff
@@ -432,6 +448,15 @@ type UdpStat struct {
 	TxQueued uint64
 }
 
+func (u *UdpStat) UdpStatMap() map[string]uint64 {
```
Comment (dashpole) [inline info/v1/container.go]: I would prefer not to add functions here.  Do you think you could just create the list within the prometheus function like `container_tasks_state`?  Same goes for the TcpStatMap function

**Comment by dashpole** [comment]
looks good.  please squash your commits

**Comment by Colstuwjx** [comment]
Done.

**Comment by Colstuwjx** [comment]
I've already update to date, let me know if there is anything need to correct, @dashpole .

**Pull request merged**